gallery.php

```diff
@@ -335,7 +335,7 @@
   </div>
 </body>
 <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
-<script src="http://netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script>
+<!--<script src="http://netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script> -->
```

Comment:
The code is correct.There was problem with the script.When I removed it the
drop down worked perfectly.
On Sep 24, 2015 7:05 PM, "Siddharth Kannan" <notifications@github.com>
wrote:

> In gallery.php
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/60#discussion_r40318707>
> :
>
> > @@ -335,7 +335,7 @@
> >    </div>
> >  </body>
> >  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
> > -<script src="http://netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script>
> > +<!--<script src="http://netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script> -->
>
> @vinaychandil <https://github.com/vinaychandil> Did commenting out this
> line actually do the trick? Is bootstrap included somewhere else in the
> same file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/60/files#r40318707>
> .
>